docs/src/pages/components/selectfield/examples/SelectFieldOptionsExample.tsx

```diff
@@ -0,0 +1,22 @@
+import { SelectField, Flex } from '@aws-amplify/ui-react';
+
+export const SelectFieldOptionsExample = () => (
+  <Flex direction="column">
+    <SelectField
+      label="Animals"
+      options={['lions', 'tigers', 'bears']}
+    ></SelectField>
+
+    <SelectField label="This is the same as the example above">
+      <option value="lions" label="lions">
```

Comment:
How about if we map through the array `[lions, tigers, bears]`?

Reply:
We could do that. The reason I did it this way was to explicitly show the output of the `options` prop, which does the mapping.

Reply:
I like it how it is, since it's here for demonstrating the equivalent example without using the `options` prop.

Reply:
Oh, I see. I didn't pay attention to line 4-8